packages/components/src/flyout/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as Flyout } from './flyout';
+// TODO: should the Content component be exported?
```

Comment:
Should the `FlyoutContent` component be exported?

Reply:
No. I'd argue that because it's just an internal component it shouldn't even be exported here either.